### PR TITLE
Create a token updates

### PR DESCRIPTION
**Pull request by mikemaccana**
We used `creating a token` to film last week, here's lots of small fixes:

 - Grammarly run over everything
 - Add 'token' as a keyword, remove 'web3js' (this tutorial doesn't never used web3js)
 - Get people to look at and use Explorer more, and add screenshots to show what they can expect.
 - Focus on faucet.solana.com for getting SOL per Jacob's recommendation.
 - Use a better name than `key` as the name of the keypair for the mint authority. The other key is also a key! We've picked `bos` for 'boss'.
 - Use a list and alphabetically sort the list of online storage services
 - Bundlr is now Irys
 - Talk about the image first, then the metadata JSON, since the meatdata JSON must include the image.

### CODE REVIEW

File: content/guides/getstarted/how-to-create-a-token.md [hunk 1]
```diff
@@ -22,270 +22,261 @@ keywords:
...
 
-We will also set the Solana CLI to use devnet for this example:
+![Solana Faucet](/assets/guides/make-a-token/solana-faucet.png)
+
+For Solana devnet we will just be able to get some free SOL,
+[the DevNet faucet](https://faucet.solana.com) If you get rate limited, follow
+this
+[guide to get devnet SOL](/content/guides/getstarted/solana-token-airdrop-and-faucets.md).
+Remember to fund your mint authority account - i.e. the keypair that starts with
+`bos` in your `new-token` folder. You can also just run:
```
Comment (ZYJLiu) [inline content/guides/getstarted/how-to-create-a-token.md:120]:
```suggestion
`bos` in your `new-token` folder. To get your address, you can also just run:
```

File: content/guides/getstarted/how-to-create-a-token.md [hunk 2]
```diff
@@ -22,270 +22,261 @@ keywords:
...
-`https://api.devnet.solana.com`. From now on all commands will be executed on
-the devnet. To release your token on mainnet you will need to perform all step
-from here again but set the cluster to mainnet using `-um` flag instead of
-`-ud`.
+For mainnet you will need to fund the account with actual SOL. You can buy SOL
+with regular currency in places like centralized exchanges, crypto on-ramps, or
+swap other tokens for SOL on decentralized exchanges.
 
-## Fund the account
+## Make a Mint Address (`mnt`)
```
Comment (ZYJLiu) [inline content/guides/getstarted/how-to-create-a-token.md:130]:
```suggestion
## Make a Mint Address
```

The url gets parsed as object-object:
`/how-to-create-a-token#make-a-mint-address-object-object`

File: content/guides/getstarted/how-to-create-a-token.md [hunk 3]
```diff
@@ -22,270 +22,261 @@ keywords:
...
 
 ```bash
-spl-token create-token --program-id TokenzQdBNbLqP5VEhdkAS6EPFLC1PHnBqCXEpPxuEb \
- --enable-metadata niceG6oxHmPcXVdNaUoECzWXn8Jz8fA5Q99QauJ4Gun.json
+spl-token create-token --program-id TokenzQdBNbLqP5VEhdkAS6EPFLC1PHnBqCXEpPxuEb --enable-metadata mntsBJFZE48XnQteT1vekH35UXDfamtdpFv6Xh8A7Gy.json
 ```
 
-This is also where you would be able to add additional
+<Callout type="note">
+This is also time time to add additional
```
Comment (ZYJLiu) [inline content/guides/getstarted/how-to-create-a-token.md:173]:
```suggestion
This is also time to add additional
```

File: content/guides/getstarted/how-to-create-a-token.md (hunk 2 above)
Comment (mikemaccana) [reply content/guides/getstarted/how-to-create-a-token.md]: Good spot!

**Review by ZYJLiu — APPROVED** [review]

**Pull request merged**